libr/core/cbin.c

```diff
@@ -1694,8 +1699,8 @@ static int bin_relocs(RCore *r, int mode, int va) {
 		r_cons_println ("[Relocations]");
 		r_table_set_columnsf (table, "XXss", "vaddr", "paddr", "type", "name");
 	} else if (IS_MODE_JSON (mode)) {
-		// start a new JSON object
-		pj = pj_new ();
+		const ut8 *encoding = r_config_get(r->config, "json.encoding");
```

Comment:
same here, write a separate function that takes RCore and constructs this pj instance with the right settings as @ret2libc said